app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -288,6 +291,18 @@ def create_vm(args = {})
       vm
     end
 
+    def id_name_substitution(argument_list, argument)
+      argument_list.detect { |a| a.name == argument }.try(:id) || argument
+    end
+
+    def verify_argument(argument_list, argument)
```

Comment:
how about get_verified_argument?

Reply:
I agree it is a better name, I will change it.